contents/blog/raising-3m-for-os.md

```diff
@@ -0,0 +1,230 @@
+
+![VC Hype Cycle](../images/vc-hype-cycle.jpg)
+
+Especially early, VC is buying into your potential value. This makes sense - if your company has a 1% chance of being worth $1Bn or more, it is rational for people to take that bet if the price is right.
+
+The downside of course is that you risk hype<>reality disonance! If you can't achieve results with the money you raised, whilst increasing your burn rate, you are at the mercy of those funding you if you will survive. Not cool, but you took this route!
+
+You also have investors, who have given you lots of money. It's important not to fall in love with them for this reason and to forget to prioritize the team and customers, hopefully in that order. Do that, and your investors will be very happy :)
+
+Oh, and you can't pay yourself a huge dividend if you succeed as a way of taking money out of the company, although it might become possible to sell some shares later if you're rocketing (perhaps to help you keep taking risk), and in the short run, you can now probably afford to pay yourself something - hoorah. Going for VC means you are committing to an exit or a failure - you can't really change your mind later that you want to take things more steadily.
+
+Turning your project into a VC backed company means you will have to spend a lot more time dealing with people. Investors, a larger team, probably more customers, and probably a larger community if you succeed in growing. Depending on your personality, this might be a good or a bad thing.
+
+PostHog chose this route. When we're 80, we think we want to have taken a couple of big swings in our lives even if we don't own the whole pie. On the product side, we can build something open source that will outlast us if we get it wrong, and we think the skills and lessons it will give us will help us in our future careers anyway.
+
+## Bootstrapping
+
+By this we mean getting real customers to pay you money and fuelling your own growth at whatever pace you see fit! Basecamp have a great [philosophy](https://basecamp.com/books/calm) on this. 
+
+You get complete control this way. That might make it easier to do the "right thing", or to be more creative in what you build since you won't have to be disrupting or creating a multi billion dollar market as a constraint. However, the financial pressure of payroll in the early days might make it tough to optimize for long run decisions. You need to feed yourself somehow!
+
+We think there is also a lot of risk in this route too. Humans get tired and hungry. It takes a ton of drive to achieve a profitable company. Giving up would be the major risk. We felt this was a similarly high level of risk to inflating your burn rate above your revenue. That risk is likely worse with open source rather than SaaS - you'll have to manage to build a free product first, then a paid version (unless you go down the hosted/not hosted route, which can create competition from the cloud providers).
```

Comment:
I think "you need to manage to build a free product first, then a paid version" should be more of a thread throughout the article, rather than mentioned in passing, as it's a huge difference between normal SaaS companies